presets/mainnet/eip7594.yaml

```diff
@@ -4,3 +4,7 @@
 # ---------------------------------------------------------------
 # `uint64(2**6)` (= 64)
 FIELD_ELEMENTS_PER_CELL: 64
+# uint64(floorlog2(get_generalized_index(BeaconBlockBody, 'blob_kzg_commitments'))
+KZG_COMMITMENTS_INCLUSION_PROOF_DEPTH: 4
```

Comment:
are these different from `KZG_COMMITMENT_INCLUSION_PROOF_DEPTH` values in deneb?